smt/expr.cpp

```diff
@@ -1058,6 +1083,7 @@ expr expr::subst(const expr &from, const expr &to) const {
 }
 
 void expr::printUnsigned(ostream &os) const {
+  // TODO: handle fp values
```

Comment:
I don't think these functions will be able to support floats, nor is probably desired?

Reply:
No, I'll add pretty printer for fp constants in another function.